packages/react/src/components/Masthead/README.stories.mdx

```diff
@@ -242,6 +242,16 @@ const platformData = {
 <Masthead platform={platformData} navigation="default" />;
 ```
 
+## searchAlwaysOpen
+
+Search will always be open, it cannot be closed.
+Clicking the close button will clear and focus the search input only.
+> This should only be used when there are no navigation items in the masthead.
+
+```javascript
+<Masthead navigation="default" searchAlwaysOpen />;
+```
```

Comment:
So I'm wondering if there was a disconnect after our conversation the other day, and maybe it's a misunderstanding on my part of the need. The `searchAlwaysOpen` needs design exploration on how to handle that `clear`/`close` user interaction. Are we ok with the `x` icon to clear when other places of the system it's used to `close`, or should we have differential indicator. The way I wrote the functional specs is if the `openOnLoad` is true then `close` on `blur` shouldn't happen, and if the user closes it at any point it should retain the user's value.

Reply:
No disconnect on my part, this is exactly the behavior I proposed. The purpose of this draft is so the design team can review this option. If a different icon should be used to clear the field, we can change that  (or maybe something like a tooltip that says `Clear` when it doesn't close the input).

For `searchOpenOnload`, an accessibility issue was opened a little while back that required it to close on `blur`, I'll have to go digging to find it. But if that's still the intended behavior we should probably track that in a separate issue.

Reply:
Right this is what you proposed, but we haven't made any decisions/need to move forward with that proposal just yet. No designs, or functional specs around it yet.